dashboard/app/views/levels/editors/fields/_instructions.haml

```diff
@@ -4,52 +4,38 @@
     Instructions
 
   #instructions.in.collapse
-
-    %p
-      Every level can have a combination of short and long instructions. When both
-      are present, the long instructions will be presented by default but the
-      instructions component can be "collapsed" down to show just the short
-      instructions. This allows us to provide both detailed, potentially verbose
-      instructions in the long instructions and short reminders of level goals
-      that don't take up too much screen space in the short instructions.
-
-    %p
-      If only one version of instructions is provided, only that version will be
-      shown.
-
-    %p
-      If neither instructions field is provided, the entire instructions component
-      (including the hint prompt) will be omitted.
-
-    .field
-      = f.label :short_instructions, "Short Instructions"
-      ~ f.text_area :short_instructions,
-        placeholder: 'some short, plaintext instructions, used to quickly communicate the goals of the level without taking up too much vertical real estate',
-        rows: 4, class:"input-block-level"
+    - unless (@level.is_a?(Applab) || (@level.is_a?(Gamelab) && !@level.is_a?(GamelabJr)) || @level.is_a?(Weblab))
```

Comment:
FYI, you can use `@level.instance_of?(Gamelab)` to make sure you do not match on sublcasses: https://ruby-doc.org/core-2.4.1/Object.html#method-i-instance_of-3F

Reply:
Oh great thanks!!